app/employee-app/backend/modules/authorization/types.bal

```diff
@@ -0,0 +1,33 @@
+//
+// Copyright (c) 2005-2010, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
// Copyright (c) 2005-2024, WSO2 Inc. (http://www.wso2.org).
```
Suggesting this as we are making this code open source

Reply:
ACK and resolved